src/screens/profile/components/user-info/styles.ts

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor } from 'src/common/enums';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    marginTop: 15,
```

Comment:
It's better to handle margins on the usage-level and pass `cotainerStyle` prop to the component